### ISSUE TITLE
Canais Oi Play sem HD

### ISSUE THREAD

**Issue by marcusmuller** [olavopeixoto/plugin.video.brplay#156]
Só consigo abrir canais que não tenham "HD" no final. Os outros abrem, mas é só fica em SD e as vezes travão kodi chegando ao ponto de fechar o app.

No app Oi Play, pra smartphone e também na WEB, eu consigo assistir em HD, mas no kodi não.

**Comment by marcusmuller** [comment]
Voltando aqui pra saber se alguém tem ideia do que está acontecendo com os canais Oi Play. Só está rodando canais em em SD.

**Comment by olavopeixoto** [comment]
Não estou tendo esse problema. Todos os canais estão funcionando pra mim.

Tem o log?

**Comment by marcusmuller** [comment]
Eu não estou consigo encontrar o arquivo aqui no Android de jeito nenhum. Vou tentar no PC. 

Aqui vai alguns screenshot da tela em SD

MTV LIVE HD 

![Screenshot_20210828-222833_Kodi](https://user-images.githubusercontent.com/28604497/131235392-73dc0fed-ede3-489a-8afa-ff9431d0fe95.jpg)
![Screenshot_20210828-222917_Kodi](https://user-images.githubusercontent.com/28604497/131235394-1fb0cbf7-5e16-42da-aaa6-1d1c67480d53.jpg)



BAND SPORTS HD

![Screenshot_20210828-223128_Kodi](https://user-images.githubusercontent.com/28604497/131235406-3d19c6b4-af61-4cd1-8e25-c81ce6295738.jpg)
![Screenshot_20210828-223054_Kodi](https://user-images.githubusercontent.com/28604497/131235408-61f5111c-634c-4eeb-949c-5d8381efc7f6.jpg)
![Screenshot_20210828-223103_Kodi](https://user-images.githubusercontent.com/28604497/131235412-1ed17a7f-8553-4e75-be55-1e617109f5bf.jpg)

**Comment by marcusmuller** [comment]
Testando agora no PC e o problema aqui é maior, pois o video dos canais oi nem abrem.

p.s: uso player externo VLC.
[kodi.log](https://github.com/olavopeixoto/plugin.video.brplay/files/7072593/kodi.log)

**Comment by marcusmuller** [comment]
Testei também com os diferentes Modo de Reprodução (Automático, Manual e Adaptativo) e nada!

2021-08-29 10:34:02.931 T:12752  NOTICE: Creating InputStream
2021-08-29 10:34:02.943 T:12752   ERROR: XFILE::CDirectory::GetDirectory - Error getting C:\Program Files\Kodi\addons\inputstream.adaptive\
2021-08-29 10:34:03.094 T:12752   ERROR: AddOnLog: InputStream Adaptive: Unable to load widevine shared library (C:\Users\mmcar\AppData\Roaming\Kodi\cdm\widevinecdm.dll)
2021-08-29 10:34:03.094 T:12752   ERROR: AddOnLog: InputStream Adaptive: OpenDRMSystem failed
2021-08-29 10:34:03.094 T:12752   ERROR: CVideoPlayer::OpenInputStream - error opening 

[kodi.log](https://github.com/olavopeixoto/plugin.video.brplay/files/7072604/kodi.log)

**Comment by olavopeixoto** [comment]
Os canais da Oi possuem DRM, pra funcionar somente no player interno do Kodi com o inputstream.adaptive e a biblioteca do Widevine (libwidevinecdm) instalados.
No VLC não irá funcionar.

**Comment by marcusmuller** [comment]
Certo. Mas pq na plataforma Android eu não consigo HD? Eu uso o Kodi no Fire tv Stick e ele possui a qualidade L1 pra ter conteúdo em HD.

**Comment by marcusmuller** [comment]
Vou tentar pegar o log

**Comment by marcusmuller** [comment]
Aqui está o log do Fire TV 
[kodi.log](https://github.com/olavopeixoto/plugin.video.brplay/files/7079903/kodi.log)
